### PR TITLE
Fix/autorenew

### PR DESCRIPTION
**Pull request by marie-j**
MANAGER-3199, MANAGER-3198

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-manager#2410.1 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✘

File: client/app/billing/autoRenew/actions/update/manualPayment/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "billing_autorenew_service_update_manual_sumup": "Vous avez sélectionné le renouvellement manuel pour votre service {{ serviceName }}",
   "billing_autorenew_service_update_manual_non_automatic": "Dans ce mode de renouvellement, votre service n'est pas automatiquement reconduit lorsqu'il arrive à échéance.",
-  "billing_autorenew_service_update_manual_expiration": "Plusieurs notifications vous seront envoyées par mails durant les semaines/jours précédents son terme, afin de vous prévenir de son expiration le {{ expiration }}.",
+  "billing_autorenew_service_update_manual_expiration": "Plusieurs notifications vous seront envoyées par mails durant les semaines/jours précédant son terme, afin de vous prévenir de son expiration le {{ expiration }}.",
```
Comment (antleblanc) [inline client/app/billing/autoRenew/actions/update/manualPayment/translations/Messages_fr_FR.json]: Is this text has been validated by content?

I personally prefer rename `mails` into `e-mail`.

Do you agree?

Comment (marie-j) [reply client/app/billing/autoRenew/actions/update/manualPayment/translations/Messages_fr_FR.json]: This is the text that was provided by CX in the mock ups

Comment (FredericEspiau) [reply client/app/billing/autoRenew/actions/update/manualPayment/translations/Messages_fr_FR.json]: `CX` !== `Content`, `Content` wants us to write it `e-mail` ;)

**Pull request merged**